src/powershell/Public/ConvertTo-FinOpsSchema.ps1

```diff
@@ -0,0 +1,158 @@
+    This conversion aids in data management, analytics, and reporting by ensuring data consistency and adherence to standards. 
+    It's important to note that the script is specifically designed for schema version 0.5 (as of September 2023) and may not fully support older versions.
+    Please review output and report any issues to https://aka.ms/ftk.
+
+    .PARAMETER ActualCost
+    Specifies the input cost data that will be converted. The object must adhere to a supported Microsoft Cost Management schema. Provide the filepath of the data to be converted. Example usage: ..\..\sample-data\EA_ActualCost.csv
+
+    .PARAMETER ExportAllColumns
+    Optionally specifies whether to export all columns in the input data or only the columns specified in the column mapping (Select-Object). When this parameter is set to $false, only the columns specified in the column mapping will be exported. Default value is $true, meaning all columns will be exported.
+
+
+    .EXAMPLE
+    ConvertTo-FinOpsSchema -ActualCost ..\..\sample-data\EA_ActualCost.csv  -ExportAllColumns $false | Export-Csv -Path .\EA_ActualCost_Output.csv -NoTypeInformation
+    Converts the input data found in EA_ActualCost.csv, exporting only the columns specified in the column mapping (Select-Object), and stores the converted data in EA_ActualCost_Output.csv.
+
+    .EXAMPLE
+    ConvertTo-FinOpsSchema -ActualCost ..\..\sample-data\EA_ActualCost.csv  -ExportAllColumns $true | Export-Csv -Path .\EA_ActualCost_Output.csv 
+    Converts the input data found in EA_ActualCost.csv, exporting all columns specified in the script, and stores the converted data in EA_ActualCost_Output.csv.
+
+    .LINK
+    https://aka.ms/ftk/ConvertTo-FinOpsSchema
+#>
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $ActualCost,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $ExportAllColumns = $true
+)
+
+function Get-AccountType {
+    [CmdletBinding()]
+    param(
+        [PSCustomObject]
+        $Object
+    )
+
+    $ftk_AccountType = switch ($true) {
+        { $Object.BillingAccountId -eq $Object.BillingProfileId } { "EA" }
+        { $Object.BillingAccountId.Contains(":") } { "MCA" }
+        default { "Other" }
+    }
+    
+    return $ftk_AccountType
+}
+
+
+function ConvertTo-FinOpsSchema {
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+    [CmdletBinding()]
+    param(
+        [string]
+        $ActualCost,
+
+        [bool]
+        $ExportAllColumns
+    )
+
+    if (-not (Test-Path -Path $ActualCost)) {
+        Write-Error "Input file $ActualCost does not exist."
+        return
+    }
+
+    $csvData = Import-Csv -Path $ActualCost
+    
+    $rowCount = $csvData.Count
+    $processedCount = 0
+    $start = [DateTime]::Now
+    
+    $transformedData = $csvData | ForEach-Object {
+        $object = $_
```

Comment:
Adding a few helpers. The Is*Cost properties are not implemented correctly here. I need to go look up how to set this again. This should be good for now, assuming things work correctly. We can add the actual logic later.
```suggestion
        $object = $_
        $object | Add-Member -MemberType NoteProperty -Name IsActualCost -Value $true # TODO: Filter actual dataset to only commitment purchases
        $object | Add-Member -MemberType NoteProperty -Name IsAmortizedCost -Value $false # TODO: Pull amortized cost from amortized dataset
        $accountType = Get-AccountType $object
```

Reply:
done